management/external_test.go

```diff
@@ -240,7 +332,7 @@ func TestAddExternal(t *testing.T) {
 			},
 		}
 		addExternalOK, err := client.Default.External.AddExternal(params)
-		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "invalid field NodeId: value '' must not be an empty string")
+		pmmapitests.AssertAPIErrorf(t, err, 400, codes.InvalidArgument, "runs_on_node_id and node_id should be specified together.")
```

Comment:
If comment https://github.com/percona/pmm-managed/pull/611#discussion_r553948018 is valid - here and below should be fixed as well.